### DISCUSSION TITLE
How can I create  dated note under the current note?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#6334] [Q&A]
Hi

I like the journal notes but most of the time I need them to be under certain node instead of locaalized Journal note. Basically I am looking for a way to create a dated (hopefully with cutmoized title formatting) under a current note. I actually do not mind it still creates it under Journal top note, just that it needs to properly link it below the currently open note.

thanks

**Comment by r-nd-m** [comment]
+1
